smartsim/settings/alpsSettings.py

```diff
@@ -129,6 +129,66 @@ def set_excluded_hosts(self, host_list):
             raise TypeError("host_list argument must be list of strings")
         self.run_args["exclude-node-list"] = ",".join(host_list)
 
+    def set_cpu_bindings(self, bindings):
+        """Specifies the cores to which MPI processes are bound
+
+        This sets ``--cpu-binding``
+
+        :param bindings: List of cpu numbers
+        :type bindings: list[int] | int
+        """
+        if isinstance(bindings, int):
+            bindings = [bindings]
+        self.run_args["cpu-binding"] = ",".join(str(int(num)) for num in bindings)
+
+    def set_memory_per_node(self, memory_per_node):
+        """Specify the real memory required per node
+
+        This sets ``--memory-per-pe`` in megabytes
+
+        :param memory_per_node: Per PE memory limit in megabytes
+        :type memory_per_node: int
+        """
+        self.run_args["memory-per-pe"] = int(memory_per_node)
+
+    def set_verbose_launch(self, verbose):
+        """Set the job to run in verbose mode
+
+        This sets ``--debug`` arg to the highest level
+
+        :param verbose: Whether the job should be run verbosly
```

Comment:
"verbosly" -> "verbosely"? (here and other times through the code)

Reply:
Fixed here and throughout code base in https://github.com/CrayLabs/SmartSim/pull/170/commits/82114ff67a5118c6ce40455fdff5519a7dcf66b6